install_config/redeploying_certificates.adoc

```diff
@@ -385,9 +385,17 @@ file:
 
 ----
 $ ansible-playbook -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-certificates.yml
+    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-master-certificates.yml
 ----
 
+[IMPORTANT]
+====
+After executing this playbook, you need to regenerate any xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[service signing certificate/key pairs] 
```

Comment:
executing should be running
certificate/key pairs should be certificates or key pairs